version.bzl

```diff
@@ -0,0 +1 @@
+VERSION = '3.17.3'
```

Comment:
for clarity, can we rename this file protobuf_version.bzl and PROTOBUF_VERSION as the var name? This makes it unambiguous in any context.

Reply:
Fixed!